js-exercises/abbreviate-string/abbreviateString.test.js

```diff
@@ -0,0 +1,16 @@
+import { abbreviateString } from './abbreviateString';
+
+describe('abbreviateString', () => {
+  it('abbreviates the following strings', () => {
+    expect(abbreviateString('Hacktober Fest')).toEqual('Hacktober F.');
+    expect(abbreviateString('Leeroy Fitzgerald Jenkins')).toEqual('Leeroy J.');
+    expect(abbreviateString('Some arbitrary string length here.')).toEqual('Some H.');
+  });
+
```

Comment:
Just a suggestion: Do write at least 1 of your own test case. It builds that habit of writing tests.